github-link.php

```diff
@@ -9,7 +9,8 @@
 Author URI: http://www.online1.hu/webdesign/
 Domain Path:       /languages
 Text Domain:       github-link
-GitHub Plugin URI: https://github.com/szepeviktor/github-link
+GitHub Plugin URI: https://github.com/jjrohrer/github-link
```

Comment:
Yike - my newness shows.  I thought a pull request would point to specific commit.  This was added later.  Reverting now.